classes/local/util/manager.php

```diff
@@ -408,8 +408,8 @@ protected function verify_capability_exists($capabilityname) {
     protected function prepare_block_files() {
 
         if (!$this->has_common_feature('capabilities')) {
-            // 'block/<blockname>:addinstance' is required.
-            // 'block/<blockname>:myaddinstance' is also required if applicable format 'my' is set to true.
+            // Block/<blockname>:addinstance is required.
+            // Block/<blockname>:myaddinstance is also required if applicable format 'my' is set to true.
```

Comment:
This does not look correct. If you want the comment starting with a capital to make travis happy, make it like
```
// Capability block/<blockname>:addinstance is required.
```